src/Symfony/Bridge/PhpUnit/README.md

```diff
@@ -0,0 +1,62 @@
+PHPUnit Bridge
+==============
+
+Utilities for PHPUnit, especially user deprecation notices management.
+
+It provides the following features:
+
+* disable the garbage collector;
```

Comment:
This is markdown, so lists must begin with some whitespaces (same below as well).